test/files/neg/t8700/Foo.java

```diff
@@ -0,0 +1,4 @@
+public enum Foo {
+    A,
+    B
+}
```

Comment:
Could add a test with enums of the form `enum Foo { A { public void foo() {} }; public void foo(); }` (related to the `ACC_ABSTRACT` change I asked about above.